packages/application/src/application/resolvers/audit-outcome-resolver.ts

```diff
@@ -0,0 +1,22 @@
+import { injectable } from 'inversify';
+import { AuditOutcome } from '@src/infrastructure/repositories/issue-details';
+import { AuditCycle } from '../services/pull-request.types';
+
+@injectable()
+export class AuditOutcomeResolver {
+  resolve(prevAudit: AuditCycle, currentAudit: AuditCycle): AuditOutcome {
+    const prevDatacap = prevAudit.datacap_amount;
+    const currentDatacap = currentAudit.datacap_amount;
+
+    switch (true) {
+      case prevDatacap === currentDatacap:
+        return AuditOutcome.GRANTED;
```

Comment:
Have you checked this with the guys who maintain the main allocator.tech page? I think it's supposed to be MATCH, not GRANTED.